tests/lib/rules/no-unused-services.js

```diff
@@ -0,0 +1,190 @@
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-unused-services');
+const RuleTester = require('eslint').RuleTester;
+
+const { getErrorMessage } = rule;
+
+const SERVICE_NAME = 'fooName';
+const message = getErrorMessage(SERVICE_NAME);
+
+//------------------------------------------------------------------------------
+// Tests
```

Comment:
Can you add a test case to ensure that `this.foo.serviceName` does not count as a usage of the `serviceName` service? (since it's inside an arbitrary property)